app/models/report.rb

```diff
@@ -99,6 +102,17 @@ def rules
     Rule.with_discarded.where(id: rule_ids)
   end
 
+  def forwardable?
+    !target_account.local?
+  end
```

Comment:
I'm not sure if this logic is correct; I think we may want to enable forwarding if the report targets a local account, but is in a thread with remote accounts?

Reply:
I think it's also not forwardable is the reporting `account` is not local, that is, if the reporting account is remote, then it's a report we've had forwarded to us.